features/exporting_shipping_data_to_api_poczta_polska.feature

```diff
@@ -34,14 +34,14 @@ Feature: Managing shipping gateway
         And it has "package_length" field set to "22"
         And it has "cod_payment_method_code" field set to "stripe_checkout"
         And it has "collect_on_delivery_form" field set to "BANK_TRANSFER"
-        And the store has a product "Chicken" priced at "$2" in "Web-US" channel
+        And the store has a product "Chicken" priced at "$2.00" in "Web-US" channel
         And customer "user@bitbag.pl" has placed 5 orders on the "Web-US" channel in each buying 5 "Chicken" products
         And the customer set the shipping address "Mike Ross" addressed it to "350 5th Ave", "10118" "New York" in the "United States" to orders
         And those orders were placed with "PocztaPolska Express" shipping method
         And set product weight to "10"
         And set units to the shipment
 
-    @ui
+    @ui @test
```

Comment:
What's the @test?

Reply:
Fixed. After upgrading to Sylius 1.13 this test was throwing an error. I checked @test to not check the other tests and forgot to delete.